Lib/System.cpp

```diff
@@ -284,7 +274,7 @@ void System::terminateImmediately(int resultStatus)
   CALL("System::terminateImmediately");
 
   onTermination();
-  _exit(resultStatus);
+  std::quick_exit(resultStatus);
```

Comment:
I am getting 

```
Lib/System.cpp:278:8: error: reference to unresolved using declaration
  std::quick_exit(resultStatus);
       ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/c++/v1/cstdlib:155:1: note: using declaration annotated with 'using_if_exists' here
using ::quick_exit _LIBCPP_USING_IF_EXISTS;
^
Lib/System.cpp:278:19: error: redefinition of 'resultStatus'
  std::quick_exit(resultStatus);
                  ^
Lib/System.cpp:273:39: note: previous definition is here
void System::terminateImmediately(int resultStatus)
```

even after having included `<cstdlib>` Googling does not help much. This is probably an issue of

```
gcc --version
Apple clang version 13.1.6 (clang-1316.0.21.2.5)
Target: x86_64-apple-darwin21.4.0
```

Reply:
`gcc --version` returning `clang` is ...interesting, but probably not the issue. I obviously can't reproduce, does `::quick_exit` compile for you?

Reply:
Well, I meant the whole clang ecosystem ;)

My `cstdlib` contains:

```
#if !defined(_LIBCPP_CXX03_LANG) && defined(_LIBCPP_HAS_QUICK_EXIT)
using ::at_quick_exit _LIBCPP_USING_IF_EXISTS;
using ::quick_exit _LIBCPP_USING_IF_EXISTS;
#endif
```

`::quick_exit` does not help.

Reply:
I'd expect `!defined(_LIBCPP_CXX03_LANG)` to be fine, so I guess it's failing on `_LIBCPP_HAS_QUICK_EXIT`...but I have no idea why, and I guess if we can't use the standards-compliant thing we should go back to using POSIX `_exit`.